### DISCUSSION TITLE
Allow custom icons & no icon

### DISCUSSION THREAD

**Discussion by SoCuul** [massCodeIO/massCode#390] [Ideas]
It would be great to be able to add custom icons (from either png/svg) as well as setting a blank icon (currently you have to manually edit the db.json file to set an invalid icon id)

**Comment by antonreshetov** [comment]
You can simply press the "Reset" button to return to the default icon.

![image](https://github.com/massCodeIO/massCode/assets/17480987/f2f75643-ba83-4e0a-903b-5ea1dcbdc90a)

**Comment by SoCuul** [comment]
Unfortunately I'm not talking about the default icon, but an "icon" that has no texture. Basically a fully transparent icon, that looks like there's no icon there.
